Add a page about the end of iteration demos session (#258)
canonical-web-and-design/practices: team/end-of-iteration-demos.md

---
title: End of iteration demos
description: Time to celebrate the great work completed at the end of the iteration
---

> Our teams demo session is an hour on the last day of the iteration. This is a chance to share and celebrate the amazing work done during the iteration. The purpose of this meeting is to give some time to look back at the work completed and to share it with the rest of the team. It's key that everyone is aware of what each other are working on so we don't design and develop in silos.

Reviewer: ```suggestion
Our team's demo session is an hour on the last day of the iteration. This is a chance to share and celebrate the amazing work done during the iteration. The purpose of this meeting is to give some time to look back at the work completed and to share it with the rest of the team. It's key that everyone is aware of what each other are working on so we don't design and develop in silos.
```

> ## Demos completed work

Reviewer: ```suggestion
## Demo completed work
```

> A demo should only be completed work or something you are confident will be complete by the end of the iteration. As when carried forward to the next iteration it will be tempting to demo again. If you are unsure refer to the [backlog sheet](https://docs.google.com/spreadsheets/d/1HH7Elz5ayh4sjH6avh0Z8_4riqlJsZ9JNMxxlesH8Go/edit) for a list of completed epics.

Reviewer: ```suggestion
A squad should only demo completed work or work you are confident will be complete by the end of the iteration. This is because it is too tempting to share the work again in the next demo.

 If you are unsure refer to the [backlog sheet](https://docs.google.com/spreadsheets/d/1HH7Elz5ayh4sjH6avh0Z8_4riqlJsZ9JNMxxlesH8Go/edit) for a list of completed epics and ask other people on your squad for advice.
```

---
title: End of iteration demos
description: Time to celebrate the great work completed at the end of the iteration
---

> Our team's demo session is an hour on the last day of the iteration. This is a chance to share and celebrate the amazing work done during the iteration. The purpose of this meeting is to give some time to look back at the work completed and to share it with the rest of the team. It's key that everyone is aware of what each other are working on so we don't design and develop in silos.

Reviewer 2: ```suggestion
Our team's demo session is an hour on the last day of the iteration. This is a chance to share and celebrate the amazing work done during the iteration. The purpose of this meeting is to give some time to look back at the work completed and to share it with the rest of the team. It's key that everyone is aware of what everyone else is working on so we don't design and develop in silos.
```

> ## Demo completed work

Reviewer 3: I'm not crazy about the idea of enforcing this. I think we should always encourage people to share their work, and demo anything they're proud of.

I don't really see any downsides to allowing people to demo anything, but if we do get issues with, for example, people demoing the same work over and over again, I'd rather we address it after we have the problem rather than trying to preempt the problem.

Author: Well this was the main point of this document and so it seems to agree on this point as a team before this can be committed too.

A squad should only demo completed work or work you are confident will be complete by the end of the iteration. This is because it is too tempting to share the work again in the next demo.

> If you are unsure refer to the [backlog sheet](https://docs.google.com/spreadsheets/d/1HH7Elz5ayh4sjH6avh0Z8_4riqlJsZ9JNMxxlesH8Go/edit) for a list of completed epics and ask other people on your squad for advice.

Reviewer 3: I'm a bit worried this link will be redundant far quicker than we tend to update practice documents. Maybe don't link? Or hedge it by saying "the backlog (in [this spreadsheet] at the time of writing)" or something.

Author: I would hope that if we switch to another sheet we would add a note and link at the top for clarity.
